core/src/main/resources/jenkins/install/SetupWizard/authenticate-security-token.jelly

```diff
@@ -16,22 +16,17 @@
                   
                     <h1>${%Unlock Jenkins}</h1>
                     <p>
-                        ${%To ensure Jenkins is securely set up by the administrator, a setup security token has been printed to the logs.}
-                        <small>
-                            <a href="https://jenkins-ci.org/redirect/find-jenkins-logs" target="_blank">
-                                ${%Not sure where to find the logs?}
-                            </a>
-                        </small>
+                        ${%jenkins.install.findSecurityTokenMessage(it.initialAdminPasswordFile)}
                     </p>
-                    <p>${%Please copy the token and paste it below.}</p>
+                    <p>${%Please copy the password and paste it below.}</p>
                     <j:if test="${error}">
                         <div class="alert alert-danger">
                           <strong>${%ERROR:} </strong>
-                          ${%There is a problem with the security token, please check the logs for the correct token}
+                          ${%The password entered is incorrect, please check the file for the correct password}
```

Comment:
Given how often the wording of these messages change, maybe it makes sense to move them into the .properties file?